modules/templates/helper.go

```diff
@@ -342,3 +343,33 @@ func DiffLineTypeToStr(diffType int) string {
 	}
 	return "same"
 }
+
+// TrN returns key to be used for translation depending on count
+func TrN(lang string, cnt interface{}, key1, keyN string) string {
+	var c int64
+	if t, ok := cnt.(int); ok {
+		c = int64(t)
+	} else if t, ok := cnt.(int16); ok {
+		c = int64(t)
+	} else if t, ok := cnt.(int32); ok {
+		c = int64(t)
+	} else if t, ok := cnt.(int64); ok {
+		c = t
+	} else {
+		return keyN
+	}
+
+	// Special rules for Latvian language
+	if lang == "lv-LV" {
```

Comment:
Maybe outside scope of this PR, but...

I'm not sure having hard-coded language-specific rules in the code is the right approach. I think ideally that language-specific rules should be expressed in that language's locale file, not in code. Two reasons for this:
1. Many gitea translators are not familiar with the codebase
2. Even hard-coded rules like this one may not be enough to support all languages. For instance, Arabic has three grammatical numbers (singular, dual, plural), so a function taking `key1` and `keyN` arguments wouldn't work for Arabic, regardless of what hard-coded rules we add.

Of course, encoding language-specific rule in the locale may require a more expressive i18n scheme, but that seems like the more sustainable approach.

Reply:
@ethantkoenig I don't see how such rule could be implemented in locale file. That would require to implement math parser in GoLang so that such rules could be used and this is totally out of scope for this PR